core/tests/test_utils.py

```diff
@@ -140,12 +157,26 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Adds a suffix to the UNICODE_TEST_STRING
+
+        Args:
+            suffix: the suffix pattern to the UNICODE_TEST_STRING
+
+        Returns:
+            string with UNICODE_TEST_STRING suffixed with string `suffix`
+        """
         return '%s%s' % (self.UNICODE_TEST_STRING, suffix)
 
     def setUp(self):
+        """To be overridden in inherited Test classes for initialization of the
+        fixture for the test suite.
+        """
         raise NotImplementedError
 
     def tearDown(self):
+        """To be overridden in inherited Test classes to do clean up of the
```

Comment:
Cleans up the fixture after the test runs. Subclasses of TestBase should override this method.